benchmark/streamly-benchmarks.cabal

```diff
@@ -144,6 +149,7 @@ library
 
 common bench-options
   import: compile-options, optimization-options, bench-depends
+  -- XXX The use of -retsopts is for runtime?
```

Comment:
yes?

Reply:
Cool

Reply:
Just to be clear about the question,
```
ghc-options: +RTS ... -RTS
```
Limits the memory for ghc itself while compilation.
Whereas,
```
ghc-options: -rtsopts +RTS ... -RTS
```
is for limiting the memory when the executable is running.
@pranaysashank Is my understanding correct?